src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
Try testing that on TestSwarm then. :-) This would be needlessly complicated IMO.

Our support tests currently are either a boolean property or a function that returns a boolean (and that result is stable, i.e. it always returns the same value in the same browser). Complicating that logic would be confusing.

I think the separation to:

1. a test checking if the bug exists
2. its application where necessary

is a good thing to have.